app/src/main/java/xterminators/spellingbee/cli/CLIController.java

```diff
@@ -28,6 +32,8 @@ public class CLIController {
     private File rootsDictionaryFile;
     /** The view which displays output and data to the user. */
     private CLIView view;
+
+    private HelpData helpData;
```

Comment:
I don't know that the controllers need to store HelpData, it can be retrieved when needed by `Puzzle.getHelpData()`.